.ci/run_tests.sh

```diff
@@ -23,7 +30,52 @@ if [ "$TEST_SUITE" == "dbless" ]; then
                      spec/02-integration/04-admin_api/15-off_spec.lua
 fi
 if [ "$TEST_SUITE" == "plugins" ]; then
-    eval "$TEST_CMD" spec/03-plugins/
+    set +e
+    rm -f .failed
+
+    for p in spec/03-plugins/*; do
+        echo
+        cyan "--------------------------------------"
+        cyan $(basename $p)
+        cyan "--------------------------------------"
+        echo
+
+        $TEST_CMD $p || echo "* $p" >> .failed
+    done
+
+    cat kong-*.rockspec | grep kong- | grep -v sidecar | grep -v zipkin | grep "~" | while read line ; do
+        REPOSITORY=`echo $line | sed "s/\"/ /g" | awk -F" " '{print $1}'`
+        VERSION=`luarocks show $REPOSITORY | grep $REPOSITORY | head -1 | awk -F" " '{print $2}' | cut -f1 -d"-"`
+        REPOSITORY=`echo $REPOSITORY | sed -e 's/kong-prometheus-plugin/kong-plugin-prometheus/g'`
+        REPOSITORY=`echo $REPOSITORY | sed -e 's/kong-proxy-cache-plugin/kong-plugin-proxy-cache/g'`
+
+        echo
+        cyan "--------------------------------------"
+        cyan $REPOSITORY $VERSION
+        cyan "--------------------------------------"
+        echo
+
+        git clone https://github.com/Kong/$REPOSITORY.git --branch $VERSION --single-branch /tmp/test-$REPOSITORY
+        cp -R /tmp/test-$REPOSITORY/spec/fixtures/* spec/fixtures/ || true
+        pushd /tmp/test-$REPOSITORY
+        luarocks make
+        popd
+
+        $TEST_CMD /tmp/test-$REPOSITORY/spec/ || echo "* $REPOSITORY" >> .failed
+
+        rm -rf /tmp/test-$REPOSITORY
```

Comment:
```suggestion
```

Initially this existed because I recycled the directory for the next iteration. Now that we're not (which is better) let's not nuke the plugin directory as it's handy for when debugging locally?

Reply:
works for me!